static/js/screenly-ose.coffee

```diff
@@ -232,7 +234,12 @@ API.View.AddAssetView = class AddAssetView extends Backbone.View
   updateFileUploadMimetype: (filename) => @updateMimetype filename
   updateMimetype: (filename) =>
     mt = get_mimetype filename
+    if mt
+      console.log "MT!"
```

Comment:
Is this debug logging? I think we don't need to commit that, if someone needs it they can add it when they do.